src/components/BlogSubscribe.tsx

```diff
@@ -0,0 +1,72 @@
+import { motion } from 'framer-motion';
+
+export const BlogSubscribe = () => {
```

Comment:
This is mostly a copy of NewsLetter.tsx.  It's possible we can/should share more.